configs/projects/puppet-agent.rb

```diff
@@ -241,7 +241,14 @@
   end
   proj.component "ruby-shadow" unless platform.is_aix? || platform.is_windows?
   proj.component "ruby-augeas" unless platform.is_windows?
-  proj.component "openssl"
+
+  if ENV["USE_SYSTEM_OPENSSL"]
```

Comment:
I'm not sure this is quite doing what you want. This will evaluate to `true` even if USE_SYSTEM_OPENSSL is set to false, which seems like it could lead to unintended results.

```
morgan@river.corp.puppetlabs.net % FOO=true BAR=false irb                                                                               (git)-[remove_utopic]
irb(main):001:0> if ENV['BAR']
irb(main):002:1> puts 'true'
irb(main):003:1> end
true
```